File: java/TuduManager/domain/src/main/java/net/seymourpoler/tudumanager/domain/todo/update/models/Todo.java
package net.seymourpoler.tudumanager.domain.todo.update.models;

import java.time.LocalDateTime;

public class Todo {
    private Integer id;
    public Integer id(){ return id;}
    public String title;
    public String title(){ return title; }
    public String description;
    public String description(){return description; }
    public LocalDateTime updatedDate;
    public LocalDateTime updatedDate(){ return updatedDate; }

    public Todo(Integer id, String title, String description) {
        this.id = id;
        this.title = title;
        this.description = description;
    }

    public Todo(Integer id, String title)  {
        this.id = id;
        this.title = title;
        this.description = "";
    }

    public void update(String title, String description) {
        this.title = title;
        this.description = description;
        this.updatedDate = LocalDateTime.now();
    }
}

File: dotnet/react.mvc-core.postgres.dapper/TuduManager/src/TuduManayer.Domain/Todo/Update/Models/Todo.cs
namespace TuduManayer.Domain.Todo.Update.Models
{
    public class Todo
    {
        public int Id { get; }
        public string Title { get; private set; }
        public string Description { get; private set; }
        
        public Todo(int id, string title, string description)
        {
            Id = id;
            Title = title;
            Description = description;
        }

        public void Update(string title, string description)
        {
            Title = title;
            Description = description;
        }
    }
}